### PR TITLE
Hacking on m1 support with Chris, Harry, Harshad, Patrick

### PR DESCRIPTION
**Pull request by justaddcoffee**

### CODE REVIEW

**Comment by justaddcoffee** [comment]
This PR, in combination with [these other steps that @caufieldjh @hrshdhgd @cmungall put together](https://docs.google.com/document/d/1SAylKLCTnqf0TTPFWJNT_NAWwSXlwETG6WWp9Mdptn0/edit):

- go get an arm-compliant version of python using conda
- `conda create --name oakx-grape-env python=3.9`
- `conda activate oakx-grape-env`
- `pip install poetry`
- `poetry install`
- [Install nvm + npm](https://dev.to/ms314006/how-to-install-npm-through-nvm-node-version-manager-5gif) : Make sure the latest version is installed. 
- Install Graphvis: https://graphviz.org/download/
    -`brew install graphviz`
- `npm install -g obographviz`

Seems to allow oakx-grape to work on M1 macs...

File: pyproject.toml [hunk 1]
```diff
@@ -8,11 +8,14 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = "^3.9"
+bioregistry = "0.5.134"
```
Comment (cmungall) [inline pyproject.toml:11]: hmm, this sort of thing shouldn't be necessary but we can revisit

**Review by cmungall — CHANGES_REQUESTED** [review]
after tests pass, can merge

**Comment by cmungall** [comment]
can we get the install steps in a readme?

The graphviz stuff isn't necessary though

**Pull request merged**